contributing_to_docs/doc_guidelines.adoc

```diff
@@ -42,10 +38,26 @@ Please set your editor to strip trailing whitespace.
 
 [NOTE]
 ====
-The topic title, which is the first line of the document, is the only level 1 ( = ) title. Section headers within the topic must be level 2 ( == ) or lower.
+The topic title, which is the first line of the document, is the only level 1 ( = ) title. Section headers (modules) within the topic must be level 2 ( == ) or lower.
```

Comment:
This sentence confuses me. I first read that as that we should be using == Section Title in the modules, but I don't think that's true. We still use what looks like level one = Section Title, but in the assembly use the leveloffset. Is that what this is trying to get at? If so, maybe clarify here that it becomes level 2 or lower by way of the leveloffset in the include statement.

Reply:
@bergerhoffer good point. This is confusing. How is this?

"The topic title, which is the first line of the document, is the only level 1 ( = ) title. 
Section headers (modules) within the topic/assembly must be level 2 ( == ) or lower, by using leveloffsets in include statements."